typescript/infra/src/core/govern.ts

```diff
@@ -69,14 +69,27 @@ export class HyperlaneCoreGovernor {
     }
   }
 
-  protected async sendCalls(chain: ChainName) {
+  async governChain(chain: ChainName, confirm = false) {
```

Comment:
I would suggest to confirm to true by default, and override to false when we are in CI